rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/ClaimantResponseJsonMapper.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.cmc.rpa.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ClaimantResponse;
+import uk.gov.hmcts.cmc.rpa.DateFormatter;
+import uk.gov.hmcts.cmc.rpa.mapper.json.NullAwareJsonObjectBuilder;
+
+import java.time.LocalDateTime;
+import javax.json.JsonObject;
+
+@Component
+@SuppressWarnings({"LineLength"})
+public class ClaimantResponseJsonMapper {
+
+    @Autowired
+    private final AddressJsonMapper address;
+
+    public ClaimantResponseJsonMapper(AddressJsonMapper address) {
+        this.address = address;
+    }
+
+    public JsonObject map(Claim claim) {
+        ClaimantResponse response = claim.getClaimantResponse()
+            .orElseThrow(() -> new IllegalArgumentException("Missing response"));
+        String claimantEmail = claim.getSubmitterEmail();
+        LocalDateTime claimantRespondedAt = claim.getClaimantRespondedAt().orElse(null);
+        return new NullAwareJsonObjectBuilder()
+            .add("caseNumber", claim.getReferenceNumber())
```

Comment:
Can we move this keys to a constant please?

Reply:
i have managed same standard as of existing mappers.